tests/test_cli.py

```diff
@@ -190,3 +190,41 @@ def test():
     result = runner.invoke(cli, ['test'])
     assert result.exit_code == 0
     assert result.output == 'flaskgroup\n'
+
+
+def test_routes():
+    """Test Routes"""
+    def create_app(info):
+        app = Flask('routes_app')
+        blueprint = Blueprint('routes_blueprint', __name__)
+
+        @blueprint.route('/users')
+        def users_route():
+            pass
+
+        @blueprint.route('/users/<user_id>', methods=['GET', 'DELETE'])
+        def user_id_route():
+            pass
+
+        @blueprint.route('/posts/<post_id>', methods=['GET', 'DELETE', 'PUT'])
+        def posts_routes():
+            pass
+
+        app.register_blueprint(blueprint)
+
+        return app
+
+    @click.group(cls=FlaskGroup, create_app=create_app)
+    def cli(**params):
+        pass
+
+    runner = CliRunner()
+    result = runner.invoke(cli, ['routes'])
+
+    output = [line.strip() for line in result.output.split('\n')]
+
+    assert result.exit_code == 0
+    assert output[0] == 'routes_blueprint.posts_routes  PUT, DELETE, GET /posts/<post_id>'
```

Comment:
You aren't sorting the list of methods so you can't assume any specific order on data coming from a set.